src/pages/user/viewer/positions.jsx

```diff
@@ -7,24 +7,26 @@ import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { position_mapping_to_string } from '../../../utils/user';
 import { faTrash } from '@fortawesome/free-solid-svg-icons';
 
-const PositionList = ({ position_mappings, showUuid, reload }) => {
+const PositionList = ({ position_mappings, show_uuid , reload, canRemove }) => {
+    const demote = async (position_mapping) => {
+        if(!canRemove || !position_mapping.event_uuid) {
+            return;
+        }
+        if(window.confirm("Er du sikker på at du vil fjerne stillingen?")) {
```

Comment:
_Already discussed in PM_
Looks like we both made a function to confirm if you want to delete a position or not.
I used the "window" component that I made when creating the improved agenda system, I did also change the design of the table.

Which version should we keep?

Preview of my design:
![image](https://github.com/phoenixlan/phoenixcrew-v1/assets/30247084/9bfef8e2-c2e8-4144-96c7-77a2535c0f0e)
![image](https://github.com/phoenixlan/phoenixcrew-v1/assets/30247084/3e7e9f81-8ecd-40b4-868b-7dcebfa9fab2)

Reply:
I will merge this, and you are free to change it later.